mmvb_backend/urls.py

```diff
@@ -13,9 +13,15 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from django.contrib import admin
+from django.urls import include, path
+from django.views.generic import TemplateView
 from rest_framework.schemas import get_schema_view
 
-from ais.api.urls import router as ais_router
+from ai_implementations.api.urls import router as ai_implementations_router
+from benchmarking_sessions.api.urls import (
```

Comment:
I guess this slipped into but should not be in this PR, right?

Reply:
👀